modules/ROOT/pages/osv2-guide.adoc

```diff
@@ -23,13 +23,15 @@ Object Store version 1 and version 2. Use Studio to create an application
 that you deploy to Runtime Manager where the application runs and serves
 requests to read to and write from the object store.
 +
-*Note:* The Object Store connector's Object Store Reference
-field (objectStore-ref element in XML) must
+The Object Store Reference field for Object Store Connector (`objectStore-ref element` in XML) must
```

Comment:
`objectStore-ref element` > `objectStore-ref` element
(because `objectStore-ref` is the name of the element)

Reply:
Fixed.